vehicle-model-lifecycle/requirements.txt

```
@@ -1,4 +1,4 @@
-velocitas-lib==0.0.12
-velocitas-model-generator==0.7.1
+velocitas-lib==0.0.13
+velocitas-model-generator @ git+https://github.com/eclipse-velocitas/vehicle-model-generator@conan2
```

Replace by final version:
```suggestion
velocitas-model-generator==0.8.0
```